users_ldap_groups/users_ldap_groups.py

```diff
@@ -40,8 +40,13 @@ def contains(self, ldap_entry, attribute, value, ldap_config, company):
             (value in ldap_entry[1][attribute])
 
     def equals(self, ldap_entry, attribute, value, ldap_config, company):
-        return attribute in ldap_entry[1] and \
-            unicode(value) == unicode(ldap_entry[1][attribute])
+        if attribute in ldap_entry[1]:
+            if type(ldap_entry[1][attribute]) == type([]):
+                ldap_value = ldap_entry[1][attribute][0]
```

Comment:
to follow the semantics of equality, I think you should return `False` here if `len(ldap_entry[1][attribute]) > 1`